store/sql_store.go

```diff
@@ -110,6 +111,14 @@ func initConnection() *SqlStore {
 		}
 	}
 
+	if len(utils.Cfg.SqlSettings.DataSourceSearchReplica) == 0 {
+		sqlStore.searchReplica = sqlStore.master
```

Comment:
This should map the the replicas not master when not set.  This also makes me realize maybe this should be an array of search replicas like read replicas?